cmd/webhook/app/options/options.go

```diff
@@ -12,21 +12,23 @@ import (
 
 // Config contains the server (the webhook) cert and key.
 type Options struct {
-	CertFile       string
-	KeyFile        string
-	KubeConfigFile string
-	QPS            float32
-	Burst          int
+	CertFile          string
+	KeyFile           string
+	KubeConfigFile    string
+	QPS               float32
+	Burst             int
+	UserSkipOverwrite string
 }
 
 // NewOptions constructs a new set of default options for webhook.
 func NewOptions() *Options {
 	return &Options{
-		KubeConfigFile: "",
-		CertFile:       "",
-		KeyFile:        "",
-		QPS:            100.0,
-		Burst:          200,
+		KubeConfigFile:    "",
+		CertFile:          "",
+		KeyFile:           "",
+		QPS:               100.0,
+		Burst:             200,
+		UserSkipOverwrite: "system:serviceaccount:open-cluster-management-agent-addon:klusterlet-addon-appmgr",
```

Comment:
Since we have this open, did you want to make it support a list of users?

Reply:
It crossed my mind as well, not sure if we have a use-case for it. I'll work on having the changes ready, but not push the commit until Qiujian agrees.

Reply:
lets make it a list